ashlar/reg.py

```diff
@@ -1428,8 +1428,7 @@ def plot_layer_quality(
     if artist is 'quiver':
```

Comment:
Could you change `is` to `==` here and below while we're at it? We can merge this once that's done.